test/utils.test.ts

```diff
@@ -18,4 +19,26 @@ describe("utils", () => {
       return expect(tsUnionOf(...[0, true, "string", '"hello"'])).toBe('0 | true | string | "hello"');
     });
   });
+
+  describe("tsIntersectionOf", () => {
```

Comment:
I happened to also add a utils.test.ts before this one made it into main :D but our test cases use different approaches. Would you like me to convert my cases into the pre-existing ones?

Reply:
No, the opposite—we can move the ones there into this one. Thanks for adding these!